src/pudl/transform/eia860.py

```diff
@@ -467,15 +468,6 @@ def boiler_generator_assn(eia860_dfs, eia860_transformed_dfs):
     """
     # Populating the 'generators_eia860' table
     b_g_df = eia860_dfs["boiler_generator_assn"].copy()
-    b_g_cols = [
-        "report_year",
-        "utility_id_eia",
-        "plant_id_eia",
-        "boiler_id",
-        "generator_id",
-    ]
-
-    b_g_df = b_g_df[b_g_cols]
```

Comment:
Why does all this get excised?

Reply:
I remove this bc 🥁 I could either add in the new column that was added in this list or say *HEY* include them all bc the columns that are effectively dropped here are like.... plant name and utility name and shit we 100% want to keep because they can get slurped up in the harvesting process.

Reply:
Great